internal/go-sso/oidcauth/oidc_test.go

```diff
@@ -390,7 +390,7 @@ func TestOIDC_ClaimsFromAuthCode(t *testing.T) {
 			context.Background(),
 			state, "wrong_code",
 		)
-		requireErrorContains(t, err, "cannot fetch token")
+		requireErrorContains(t, err, "Error exchanging oidc code")
```

Comment:
Thanks for looking into this! I was thinking it would be more reliable to check a string from our internal code, so that we wouldn't have to update this every time the external library changes again.

The unit test's purpose seemed to exercise the code path for a failed exchange (checking internal message would be sufficient) and not as much to confirming the details of how the exchange failed (checking the external message would be necessary). But I might be missing the intention of the test though.

Let me know your thoughts!

Reply:
Hmm I see what you mean, can leave as is